### PR TITLE
Remoting class documentation

### PR DESCRIPTION
**Pull request by markknol**
Class descriptions

### CODE REVIEW

**Comment by markknol** [comment]
Is this any good?

File: std/haxe/remoting/LocalConnection.hx [hunk 1]
```diff
@@ -21,6 +21,9 @@
  */
 package haxe.remoting;
 
+/**
+    Allows remoting communications over a LocalConnection
```
Comment (Simn) [inline std/haxe/remoting/LocalConnection.hx]: That doesn't really say anything. What's a LocalConnection?

File: std/haxe/remoting/SocketConnection.hx [hunk 2]
```diff
@@ -22,6 +22,9 @@
 package haxe.remoting;
 import haxe.remoting.SocketProtocol.Socket;
 
+/**
+	Allows remoting communications over an XMLSocket
```
Comment (Simn) [inline std/haxe/remoting/SocketConnection.hx]: A SocketConnection in general doesn't have anything to do with XML (I hope).

**Comment by Simn** [comment]
Sorry, completely missed this one.

**Comment by Simn** [comment]
Well, some of these might require a bit of rewording but having some documentation is infinitely better than having none, so I'll go ahead and merge it.

**Comment by markknol** [comment]
I agree. Everything in std and haxe package should have documentation.

**Pull request merged**